### DISCUSSION TITLE
Welcome to visx Discussions! 📈

### DISCUSSION THREAD

**Discussion by williaster** [airbnb/visx#999] [General]
## 👋  Welcome!
  We’re enabling Discussions as a place to connect with other members of the `visx` community. We hope that you:
  * Ask questions you’re wondering about (data vis can be hard, esp if you're new to `d3`, `react`, or `TypeScript`).
  * Share ideas, examples, or inspiration.
  * Engage with other `visx` community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To kick it off, comment below with an introduction of yourself and tell us about what you do with `visx`. 

Happy charting ✌️ .

**Comment by valtism** [comment]
Heya! I'm Dan, a developer from Sydney, Australia 🏖

I'm working at [Pry](pry.co), making budgeting and finance wonderful for founders. I'm building a powerful new dashboard to visualise your finances and compare scenarios. We used to use `chart.js`, but the timely release of `xy-chart` helped make the switch. `visx` really is a breath of fresh air, and makes our custom designs so much nicer to implement.

![Dashboard](https://user-images.githubusercontent.com/1286001/103960620-ae8f9b00-51a6-11eb-9283-52e446ca1b00.png)

Thanks to the team for their hard work on this incredible library!

**Comment by jithureddy** [comment]
Hello @williaster , I am working at [Mondra](https://www.mondra.com/) and we uses the `visx` as the charting library and customized to our needs.
